lib/src/operation.rs

```diff
@@ -117,6 +118,19 @@ impl Operation {
         &self.data.metadata
     }
 
+    /// Returns true if predecessors are recorded in this operation.
+    ///
+    /// This returns false only if the operation was written by old jj.
```

Comment:
nit: since we know the current version, we might as well say "written by jj < 0.30"